beets/ui/commands.py

```diff
@@ -665,7 +664,7 @@ def show_match_tracks(self):
         """
         # Tracks.
         # match is an AlbumMatch named tuple, mapping is a dict
-        # Sort the pairs by the track_info index (at index 1 of the namedtuple)
+        # Sort the pairs by the track_info index (at index 1 of the NamedTuple)
```

Comment:
If you're adjusting this, could you clarify what **at index 1 of the NamedTuple** means? Can this be replaced by the name of the namedtuple attribute?

Reply:
I think the person who wrote this would have to explain what it means…

Reply:
@JOJ0 can you help out?